packages/workspace/src/browser/workspace-service.ts

```diff
@@ -603,10 +604,7 @@ export class WorkspaceService implements FrontendApplicationContribution, Worksp
         }
         const workspaceData: WorkspaceData = { folders: [], settings: {} };
         if (!this.saved) {
-            for (const p of Object.keys(this.schemaProvider.getCombinedSchema().properties)) {
-                if (this.schemaProvider.isValidInScope(p, PreferenceScope.Folder)) {
-                    continue;
-                }
+            for (const p of Object.keys(this.schemaProvider.getJSONSchema(PreferenceScope.Workspace).properties!)) {
```

Comment:
We now switch here from `PreferenceScope.Folder` to `PreferenceScope.Workspace`. Is this correct?

Reply:
We used to filter out the preferences that could possibly be overridden in folder scope from being saved in the workspace file (if I read the code correctly). I don't think this makes sense: if someone makes a setting in a multi-folder workspace, it should be saved into the workspace file, no?